src/server/server.rs

```diff
@@ -679,12 +684,16 @@ mod tests {
 
         let cfg = Config::new();
         let mut event_loop = create_event_loop(&cfg).unwrap();
+        let mut storage = Storage::new(Dsn::RocksDBPath(TEMP_DIR), 4096, 256).unwrap();
+        if let Err(e) = storage.start(1024) {
```

Comment:
I think it would be better to use consts defined in `config.rs` rather than numbers.